### PR TITLE
Development search and acess public services

### PR DESCRIPTION
**Pull request by deepjyotk**

### CODE REVIEW

**Comment by omerfarukbasar** [comment]
Boilerplate seems to be good!

File: public_service_finder/public_service_finder/settings.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-qkdoa^_d-cnf1)7u!pvlq1i6s9^5(__iaiczz94&e4hz1@4=26'
```
Comment (KartikeySharma) [inline public_service_finder/public_service_finder/settings.py]: Lets discuss this (Not a blocker though)

**Review by omerfarukbasar — APPROVED** [review]
Boilerplate seems good!

**Review by KartikeySharma — APPROVED** [review]

**Comment by KartikeySharma** [comment]
Approved! First PR!!

File: public_service_finder/public_service_finder/settings.py (hunk 1 above)
Comment (deepjyotk) [reply public_service_finder/public_service_finder/settings.py]: Made the changes! Thanks!

**Pull request merged**